src/components/HelpDropdown.tsx

```diff
@@ -0,0 +1,57 @@
+import { ArrowRightOutlined } from "@ant-design/icons";
+import { Button, ConfigProvider, Dropdown, MenuProps, Space } from "antd";
+import React, { ReactElement, useRef } from "react";
+import styled from "styled-components";
+
+import { DropdownSVG } from "../assets";
+import { FlexRowAlignCenter, VisuallyHidden } from "../styles/utils";
+
+const HelpButton = styled(Button)``;
```

Comment:
What does this styled component with an empty style do?

Reply:
Oops, I think this was holdover from when I'd been making tweaks to this draft PR according to Lyndsay's designs. It does nothing and will be 🔥destroyed🔥